webapp/locales/de.json

```diff
@@ -40,7 +40,7 @@
       "tagCountUnique": "Benutzer"
     },
     "invites": {
-      "description": "Einladungen sind ein wunderbarer Weg, deine Freund in deinem Netzwerk zu haben …",
+      "description": "Einladungen sind eine wunderbare Möglichkeit, Deine Freunde in Deinem Netzwerk zu haben …",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-19T22:29:06Z" title="Wednesday, February 19th 2020, 11:29:06 pm +01:00">Feb 19, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -40,7 +40,7 @@
       "tagCountUnique": "Benutzer"
     },
     "invites": {
-      "description": "Einladungen sind ein wunderbarer Weg, deine Freund in deinem Netzwerk zu haben …",
+      "description": "Einladungen sind eine wunderbare Möglichkeit, Deine Freunde in Deinem Netzwerk zu haben …",
```


It's OK for me now to use the capitalized `Du` here but according to a teacher of journalism, this spelling is antiquated and except for letters it's now common to use lowercase `du`. So for the future, I would prefer that.